contracts/InterestRates/InterestRateDebtToken.sol

```diff
@@ -0,0 +1,134 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.19;
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { Fixed256x18 } from "@tempusfinance/tempus-utils/contracts/math/Fixed256x18.sol";
+import { ERC20, ERC20Capped } from "../common/ERC20Capped.sol";
+import { IERC20Indexable } from "../Interfaces/IERC20Indexable.sol";
+import { PositionManagerDependent } from "../PositionManagerDependent.sol";
+import { IInterestRatePositionManager } from "./IInterestRatePositionManager.sol";
+
+contract InterestRateDebtToken is IERC20Indexable, ERC20Capped, PositionManagerDependent {
```

Comment:
Can't we inherit `ERC20Indexable` and just override some functions?

Reply:
Probably yes